dotcom-rendering/src/lib/braze/forceBrazeMessage.ts

```diff
@@ -1,7 +1,17 @@
+const runtimeHostname = (): string | undefined => {
+	if (typeof process !== 'undefined' && process.env.HOSTNAME) {
```

Comment:
If this works it may be a bit simpler?

```suggestion
	if (process !== undefined && process.env.HOSTNAME) {
```